README.md

```diff
@@ -31,18 +31,18 @@ The focus of the here provided scenarios is on securing the planning aspects of
 
 The execution of the concrete scenarios in the open source tools "esmini", a basic OpenSCENARIO player, and "openPASS", a simulation platform for traffic simulation, is described on Windows:
 
-_Note:_ Currently only esmini supports the OpenSCENARIO 1.1 format. For openPASS please use the [release v0.3.2](https://github.com/asam-oss/OSC-ALKS-scenarios/releases/tag/v0.3.2) with scenarios in OpenSCENARIO 1.0 format.
+_Note:_ Currently only esmini supports the OpenSCENARIO 1.3 format. For openPASS please use the [release v0.3.2](https://github.com/asam-oss/OSC-ALKS-scenarios/releases/tag/v0.3.2) with scenarios in OpenSCENARIO 1.0 format.
```

Comment:
Is it still the right tag for openPASS?

Reply:
There was no test with a newer version of openPASS, so this sentence is still valid.